doc/install.txt

```diff
@@ -69,818 +48,44 @@ The following libraries and software are optional:
     `libgpuarray`_
         Required for GPU/CPU code generation on CUDA and OpenCL devices (see: :ref:`gpuarray`.)
```

Comment:
Let's replace this line with "libgpuarray is the new backend that supports CPU/GPU code generation on CUDA and OpenCL devices. It's installation is optional (Theano can also work with the previous backend, which only supports CPU/GPU code generation on CUDA), but recommended." and delete the second note.